pkg/backup/backup.go

```diff
@@ -292,6 +297,23 @@ func (kb *kubernetesBackupper) Backup(logger logrus.FieldLogger, backupRequest *
 	return err
 }
 
+func (kb *kubernetesBackupper) writeBackupVersion(tw *tar.Writer) error {
+	hdr := &tar.Header{
+		Name:     "backup-version",
+		Size:     int64(len(buildinfo.Version)),
```

Comment:
This isn't the ark version. It needs to be a "backup version" that starts at something like 1 or 1.0.0 and then we'll manually increment as needed. And eventually have code that handles restoring different backup versions.

Reply:
Ahh, ok. So we'll just store that in a const then?